docs/integrations/amazon-aws/amazon-elastic-container-service-container-insights-cloudwatch.md

```diff
@@ -0,0 +1,643 @@
+
+2.  [Collect Logs, Metrics(Container Insights+Cloudwatch) and Traces for ECS](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/logging-using-cloudtrail.html#service-name-info-in-cloudtrail) - This version collects[ ](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/logging-using-cloudtrail.html#service-name-info-in-cloudtrail)[ECS CloudWatch Metrics](http://docs.aws.amazon.com/AmazonECS...ch-metrics.htm), [Container Insights Metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-metrics-ECS.html), [ECS Events using AWS CloudTrail](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/logging-using-cloudtrail.html#service-name-info-in-cloudtrail), Application Logs and Traces. Metrics collected by Container Insights are charged as custom metrics. For more information about CloudWatch pricing, see[ Amazon CloudWatch Pricing](https://aws.amazon.com/cloudwatch/pricing/).This solution enables you to monitor both ec2 and fargate based ecs deployments.
+
+## Collect Logs, Metrics(Container Insights+Cloudwatch) and Traces for ECS
+
+This page has instructions for collecting logs and metrics for the Amazon ECS App. It uses following data
+
+1.  Cloudwatch Metrics
+2.  Container Insights Metrics
+3.  AWS CloudTrail Events
+4.  Container Insights Performance log Events
+5.  ECS Application Logs
+6.  Traces
+
+### Creating Fields in Field Schema 
+
+Login to Sumo Logic, go to Manage Data > Logs > Fields. Search for the following fields: "**account**", "**namespace**", "**region**" field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
+
+### Creating Field Extraction Rule(s)#creating-field-extraction-rule-s) 
+
+Create Field Extraction Rule for CloudTrail Logs. Learn how to create Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
+
+Rule Name: AwsObservabilityECSCloudTrailLogsFER
+Applied at: Ingest Time
+Scope (Specific Data):
+account=* eventname eventsource "ecs.amazonaws.com"
+Parse Expression:
+| json "eventSource", "awsRegion", "requestParameters.tableName", "recipientAccountId" as eventSource, region, tablename, accountid nodrop
+| where eventSource = "ecs.amazonaws.com"
+| "aws/ecs" as namespace
+| fields region, namespace, accountid
+
+Create Field Extraction Rule for Container Insights Performance Events Logs of Task and Containers
+
+Rule Name: AwsObservabilityECSPerformanceEventsFER
+Applied at: Ingest Time
+Scope (Specific Data):
+account=* (Task OR Container)
```

Comment:
needs to have one backtick before and after -- code block

Reply:
Done

Reply:
not fixed